reV/handlers/resource.py

```diff
@@ -259,6 +260,27 @@ def __getitem__(self, keys):
 
         return out
 
+    def __iter__(self):
+        return self
+
+    def __next__(self):
+        if self._i >= len(self.datasets):
+            self._i = 0
+            raise StopIteration
+
+        dset = self.datasets[self._i]
+        self._i += 1
+
+        return dset
+
+    def __contains__(self, dset):
+        test = dset in self.datasets
+        if not test:
+            msg = "{} does not exist in {}".format(dset, self)
+            raise HandlerKeyError(msg)
```

Comment:
I think this is supposed to return False instead of raise an exception?

Reply:
hmm, all I changed was the property name...